Zend/zend_execute.c

```diff
@@ -1540,6 +1540,53 @@ static zend_never_inline void zend_assign_to_object_dim(zend_object *obj, zval *
 	}
 }
 
+static void frameless_observed_call_copy(zend_execute_data *call, uint32_t arg, zval *zv)
+{
+	if (Z_ISUNDEF_P(zv)) {
+		ZVAL_NULL(zv);
+	}
+	ZVAL_COPY_DEREF(ZEND_CALL_VAR_NUM(call, arg), zv);
```

Comment:
This code is wrong!
The original CV must not be touched. it should be kept IS_UNDEF.
Only ZEND_CALL_VAR_NUM(call, arg) should be set to IS_NULL.
Something like this:

```c
zval *dst = ZEND_CALL_VAR_NUM(call, arg);
if (Z_ISUNDEF_P(zv)) {
	ZVAL_NULL(dst);
} else {
	ZVAL_COPY_DEREF(dst, zv);
}
```